lisa/sut_orchestrator/azure/platform_.py

```diff
@@ -376,7 +376,9 @@ def _prepare_environment(  # noqa: C901
                             min_runbook.location = location_name
                             min_runbook.vm_size = azure_cap.vm_size
                             assert isinstance(min_cap.nic_count, int)
+                            assert isinstance(min_cap.disk_count, int)
```

Comment:
move the assert one line lower. Make it's near to `min_runbook.disk_count = min_cap.disk_count`